rpa-manager/modules/ROOT/pages/settings-adminsettings-general.adoc

```diff
@@ -0,0 +1,36 @@
+= Configure General Settings
+:page-notice-banner-message: MuleSoft RPA is integrating with Anypoint Platform to unify the login experience and provide support for external identity providers (IdP). After your organization is migrated to Anypoint Platform, the *Organization Management* module is replaced by the *Settings* module.
+
+The *General* tab in the *Admin Settings* view enables you to configure the currency and decimal sign to show in RPA Manager modules.
```

Comment:
Are you configuring the signs to show up in RPA manager or how the signs show up?